gtsfm/utils/metrics.py

```diff
@@ -79,19 +84,18 @@ def compute_correspondence_metrics(
             gt_scene_mesh,
             dist_threshold,
         )
-    elif gt_wTi1 is not None and gt_wTi2 is not None:
-        gt_i2Ti1 = gt_wTi2.between(gt_wTi1)
-        is_inlier, reproj_error = epipolar_inlier_correspondences(
-            matched_keypoints_i1,
-            matched_keypoints_i2,
-            intrinsics_i1,
-            intrinsics_i2,
-            gt_i2Ti1,
-            dist_threshold,
-        )
-    else:
-        return None, None
-
+        return is_inlier, reproj_error
+        
+    # no mesh is provided, so we use squared Sampson error.
```

Comment:
nit: word comments as commands, e.g., "If no mesh is provided, use squared Sampson distance."
nit: capitalize first letter of sentence

Reply:
thanks, updated.